plugin/trino-influxdb/src/test/java/io/trino/plugin/influxdb/TestInfluxTypeMapping.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.influxdb;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.testing.AbstractTestQueryFramework;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.datatype.DataSetup;
+import io.trino.testing.datatype.SqlDataTypeTest;
+import org.testng.annotations.Test;
+
+import static io.trino.plugin.influxdb.InfluxQueryRunner.createInfluxQueryRunner;
+import static io.trino.spi.type.BigintType.BIGINT;
+import static io.trino.spi.type.BooleanType.BOOLEAN;
+import static io.trino.spi.type.DoubleType.DOUBLE;
+import static io.trino.spi.type.VarcharType.VARCHAR;
+
+public class TestInfluxTypeMapping
```

Comment:
Can we add null value case to each type mapping tests?

Reply:
We can't create a single record in measurement with null value fields. it field type in measurement depends on the first record, so It only allow points as below :
![image](https://user-images.githubusercontent.com/53557579/211243632-05a30408-924b-42e1-9149-330e63059dbe.png)
for a null value in col_3, we must have to insert another points non-null value in col_3. It makes the tests confused then.